optaplanner-examples/src/main/java/org/optaplanner/examples/flightcrewscheduling/domain/Employee.java

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright 2018 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.optaplanner.examples.flightcrewscheduling.domain;
+
+import java.time.LocalDate;
+import java.util.Set;
+import java.util.SortedSet;
+
+import org.optaplanner.core.api.domain.entity.PlanningEntity;
+import org.optaplanner.core.api.domain.variable.InverseRelationShadowVariable;
+import org.optaplanner.examples.common.domain.AbstractPersistable;
+
+@PlanningEntity
```

Comment:
Is this a mistake (I don't believe so)? Because `FlightAssignment` is the `@PlanningEntity`. Maybe it's required to be able to use the `@InverseRelationShadowVariable` but even if that's the case, it's confusing.

Reply:
Yes, it's required for the @InverseRelationShadowVariable to work (that flightAssignmentSet is a shadow variable). So this class is a shadow entity.

The reasons I didn't go for a @PlanningShadowEntity class annotation is because genuine planning entity classes have both genuine variables and shadow variables.